GL_Framework/models/SomeEndpointModel.py

```diff
@@ -0,0 +1,7 @@
+from dataclasses import dataclass
+
+@dataclass
+class SomeEndpointModel:
```

Comment:
Could you please give an example of the endpoint you're expecting to create from this data model?